src/config/dom.js

```diff
@@ -0,0 +1,72 @@
+// DomApi
+//  ---------
+
+import Backbone from 'backbone';
+
+// Performant method for returning the jQuery instance
+function getEl(el) {
+  return el instanceof Backbone.$ ? el : Backbone.$(el);
+}
+
+// Static setter
+export function setDomApi(mixin) {
+  this.prototype.Dom = _.extend({}, this.prototype.Dom, mixin);
+}
+
+export default {
+
+  // Returns a new HTML DOM node instance
+  createBuffer() {
+    return document.createDocumentFragment();
+  },
+
+  // Lookup the `selector` string
+  // Selector may also be a DOM element
+  // Returns an array-like object of nodes
+  getEl(selector) {
+    return getEl(selector);
+  },
+
+  // Finds the `selector` string with the el
+  // Returns an array-like object of nodes
+  findEl(el, selector, _$el = getEl(el)) {
+    return _$el.find(selector);
+  },
+
+  // Detach `el` from the DOM without removing listeners
+  detachEl(el, _$el = getEl(el)) {
+    _$el.detach();
+  },
+
+  // Remove `oldEl` from the DOM and put `newEl` in its place
+  replaceEl(newEl, oldEl) {
+    if (newEl === oldEl) {
+      return;
+    }
+
+    const parent = oldEl.parentNode;
+
+    if (!parent) {
+      return;
+    }
+
+    parent.replaceChild(newEl, oldEl);
+  },
+
+  // Replace the contents of `el` with the HTML string of `html`
+  setContents(el, html = '', _$el = getEl(el)) {
```

Comment:
Ah interesting.. Yeah I added this because `this.el.innerHTML = undefinedVar;` puts "undefined" in the html.  But we should probably just check if html is ! undefined before acting.